### PR TITLE
Ubuntu AWS and Ubuntu pro pages

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Build of the AWS and AWS pro pages

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go to /aws and check it matches the [copy doc](https://docs.google.com/document/d/1lkH-MSPmgjCMP4gI0FW29YIKVDvBuddxA63iQhmsYIU/edit?ts=5de63dcb#) and the [design](https://usercontent.irccloud-cdn.com/file/mjMJMvDg/image.png)
- Go to /aws/pro and check it matches the [copy doc](https://docs.google.com/document/d/1pLRWJZSpE04nYt-yHBtCjxGI7VZsigcP2UJIBmR8i-0/edit#)
- Go to /aws/contact-us and check to works
- Check the thank you page at /aws/thank-you looks ok

## Issue / Card
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2030
Fixes https://github.com/canonical-web-and-design/web-squad/issues/2032

### CODE REVIEW

**Comment by anasereijo** [comment]
@anthonydillon same comments as Spencer + these ones:

Overview:

- [ ] make the AWS logo 200 px wide

- [ ] "Optimise your cloud journey with Canonical": make it 6 cols and quote 5

- [ ] quote looks very small comparing to design: make it `class="p-pull-quote"`

**Comment by anthonydillon** [comment]
@spencerbygraves @anasereijo I have addressed all the points I can for now.

>  make the "* For 18.04 LTS onwards. The maintenance period for Ubuntu Pro 14.04 LTS and 16.04 LTS is 8 years." 

I assume you mean smaller

>  The piechart is now a graph

I have removed the graph for now

**Comment by anasereijo** [comment]
@anthonydillon looks good!!

The quote on /pro page looks a bit weird, and should the quote marks be orange as usual? See below:

desktop:
Screenshot 2019-12-04 at 12 12 24

mobile:
Screenshot 2019-12-04 at 12 12 45

File: templates/aws/index.html [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">All the bits you need</h3>
+      <p>Ubuntu includes everything you need out of the box to build and operate cloud-native applications at scale. Supporting runtimes like Python, node.js and Go, and server components like Apache Kafka, NGINX, MongoDB, Redis and PostgreSQL.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
```
Comment (sowasred2012) [inline templates/aws/index.html]: Don't think this needs to be a `row`
```suggestion
  <div class="u-fixed-width">
```

File: templates/aws/pro.html [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+      </div>
+      <div class="col-5 u-align--center u-vertically-center u-hide--small">
+        <img src="https://assets.ubuntu.com/v1/a82add58-profile-aws.svg" alt="" width="200">
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
```
Comment (sowasred2012) [inline templates/aws/pro.html]:
```suggestion
  <div class="u-fixed-width">
```

File: templates/aws/pro.html [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+        </div>
+      </li>
+    </ul>
+    <p id="index-1"><small>* For 18.04 LTS onwards. The maintenance period for Ubuntu Pro 14.04 LTS and 16.04 LTS is 8 years.</small></p>
+    <p><a href="/security">Learn more about security for Ubuntu&nbsp;&rsaquo;</a></p>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
```
Comment (sowasred2012) [inline templates/aws/pro.html]:
```suggestion
  <div class="u-fixed-width">
```

File: templates/aws/pro.html [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+    <div class="col-8">
+      <h2>Broadest open source security coverage keeps CISOs sleeping at night</h2>
+      <p>Ubuntu Pro includes security coverage for the entire collection of software packages shipped with Ubuntu, tracking high and critical CVEs. Ideal for companies that have embraced open source for their new products.</p>
+      <p>Developers choose Ubuntu because it includes a set of open source packages known as ‘universe’. These include NGINX, MongoDB, Redis, Kafka&hellip; .With Ubuntu Pro, CIOs and security officers can be confident they can roll these packages out to production.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
```
Comment (sowasred2012) [inline templates/aws/pro.html]:
```suggestion
  <div class="u-fixed-width">
```

File: templates/aws/pro.html [hunk 5]
```diff
@@ -0,0 +1,267 @@
...
+          </tbody>
+        </table>
+        <p>For complete pricing, <a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-link--external">see Ubuntu Pro 18.04 LTS on the AWS Marketplace</a></p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
```
Comment (sowasred2012) [inline templates/aws/pro.html]:
```suggestion
  <div class="u-fixed-width">
```

File: templates/aws/pro.html [hunk 6]
```diff
@@ -0,0 +1,267 @@
...
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-link--external">18.04 LTS</a></h3>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821WW873" class="p-link--external">16.04 LTS</a></h3>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821V7QJP" class="p-link--external">14.04 LTS</a></h3>
+    </div>
+  </div>
+  <div class="row">
```
Comment (sowasred2012) [inline templates/aws/pro.html]:
```suggestion
  <div class="u-fixed-width">
```

File: templates/shared/forms/interactive/_aws.html [hunk 7]
```diff
@@ -0,0 +1,183 @@
...
+              <button type="submit" class="pagination__link--next p-button--positive mktoButton" aria-label="Submit">Let's discuss</button>
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
+
+    <div class="js-pagination js-pagination--4 u-hide">
+      <div class="row u-no-padding">
+        <div class="u-equal-height">
```
Comment (sowasred2012) [inline templates/shared/forms/interactive/_aws.html]: Could this class be added to the div above? `col-`s should be direct children of `row`s.

File: templates/aws/pro.html (hunk 2 above)
Comment (anthonydillon) [reply templates/aws/pro.html]: I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

File: templates/aws/pro.html (hunk 3 above)
Comment (anthonydillon) [reply templates/aws/pro.html]: I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

File: templates/aws/pro.html (hunk 4 above)
Comment (anthonydillon) [reply templates/aws/pro.html]: I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

File: templates/aws/pro.html (hunk 5 above)
Comment (anthonydillon) [reply templates/aws/pro.html]: I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

File: templates/aws/pro.html (hunk 6 above)
Comment (anthonydillon) [reply templates/aws/pro.html]: I prefer to use Vanilla as [it is documented](https://docs.vanillaframework.io/patterns/grid/). It is confusing to switch use with multiple people working on the project.

File: templates/shared/forms/interactive/_aws.html (hunk 7 above)
Comment (anthonydillon) [reply templates/shared/forms/interactive/_aws.html]: Good point

File: templates/aws/index.html (hunk 1 above)
Comment (anthonydillon) [reply templates/aws/index.html]: Same as below

**Comment by anthonydillon** [comment]
@anasereijo @sowasred2012 addressed your comments

File: templates/aws/pro.html [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+  </div>
+  <div class="row p-divider">
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 18.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
+      <p><a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 16.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
```
Comment (sowasred2012) [inline templates/aws/pro.html]: The copy doc has "The most commonly deployed Ubuntu LTS on the cloud today, Ubuntu 16.04 LTS released in 2016 with the 4.4 kernel, and is covered with Ubuntu Pro through 2024." for this section.

File: templates/aws/pro.html [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+      <p><a href="https://aws.amazon.com/marketplace/pp/B0821WW873" class="p-button--positive"><span class="p-link--external">Launch now</span></a></p>
+    </div>
+    <div class="col-4 p-divider__block">
+      <h3 class="p-heading--four">Ubuntu Pro 14.04 LTS</h3>
+      <p>The latest LTS release includes the 4.15 kernel out of the box, with an optimised 5.3 kernel available soon. Ubuntu Pro provides extended coverage through 2028.</p>
```
Comment (sowasred2012) [inline templates/aws/pro.html]: The copy doc has "Ubuntu 14.04 LTS shipped in 2014 with the 3.13 kernel. Ubuntu Pro extends our security coverage to 2022, giving you plenty of time to prepare and upgrade."

**Comment by anasereijo** [comment]
nice

**Comment by anthonydillon** [comment]
@sowasred2012 thanks updated

**Pull request merged**